pandas/tests/dtypes/test_dtypes.py

```diff
@@ -1212,3 +1212,8 @@ def test_multi_column_dtype_assignment():
 
     df["b"] = 0
     tm.assert_frame_equal(df, expected)
+
+
+def test_empty_object_array_is_string_dtype():
+    # GH #54661
```

Comment:
nitpick can you remove the space after GH.  i like the pattern GH#12345 bc it is something i grep for

Reply:
Done. I agree, it's better to have all GH refers look alike